migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -134,7 +136,20 @@ public void saveProjects(Collection<Project> projects) throws IOException {
     User user = serviceManager.getAuthorizationManager().getCurrentUser();
     for (Project project : projects) {
       project.setSecurityProfile(new SecurityProfile(user));
-      serviceManager.getProjectDao().save(project);
+      // Make sure there's a study
+      if (project.getStudies() == null) project.setStudies(new HashSet<Study>());
+      if (project.getStudies().isEmpty()) {
+        Study study = new StudyImpl(project, user);
+        study.setAlias(project.getShortName() + " study");
+        study.setDescription("");
+        study.setStudyType("Other");
+        study.setLastModifier(user);
+        project.getStudies().add(study);
+      }
+      project.setId(serviceManager.getProjectDao().save(project));
+      for (Study study : project.getStudies()) {
+        study.setId(serviceManager.getStudyDao().save(study));
```

Comment:
Maybe... More importantly, permissions would need to be inherited if Studies were provided in the migration data. Fixed